src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  const robot = (n) => {
```

Comment:
Looks good to me =) Just small amendment on checklist p2:
```suggestion
  return (n) => {
```

Reply:
Also, why do we name the function `robot`?
Let's use better naming